moved/src/move_execution/eth_token.rs

```diff
@@ -87,6 +96,26 @@ impl BaseTokenAccounts for MovedBaseTokenAccounts {
         )
     }
 
+    fn refund_gas_cost<G: GasMeter>(
+        &self,
+        to: &AccountAddress,
+        amount: u64,
+        session: &mut Session,
+        traversal_context: &mut TraversalContext,
+        gas_meter: &mut G,
```

Comment:
I think it can create the `free_gas_meter` internally inside the function, no need to take in as an input.

Reply:
That's definitely more sleek, thanks for the suggestion! Did see it in the codebase, but forgot by the point of writing the code.